website/docs/sdk-reference/js-ssr.md

```diff
@@ -602,6 +640,14 @@ for (const key of snapshot.getAllKeys()) {
 }
 ```
 
+:::caution
+Please be aware that creating and using a snapshot will not update the local cache when the cached config data is expired.
+Also, if you use [shared caching](/docs/advanced/caching/#shared-cache), it will not make the SDK synchronize with
+the external cache either (since snapshots capture the config instance that is kept in the memory by the client).
+Because of this, it is best to use snapshots combined with Auto Polling mode. In this case the SDK takes care of refreshing
+the local cache in the background. In other polling modes, you need to manually refresh it by calling `forceRefreshAsync`.
+:::
```

Comment:
If it is not too late to change it, my recommendation:

Please note that when you create and utilize a snapshot, it won't refresh your local cache once the cached configuration data expires. Additionally, when working with [shared caching](https://chat.openai.com/docs/advanced/caching/#shared-cache), creating a snapshot also doesn't trigger a sync with the external cache, since the snapshot only captures the configuration instance stored in the client's memory. Therefore, it's recommended to use snapshots in conjunction with the Auto Polling mode, where the SDK automatically updates the local cache in the background. For other polling modes, you'll need to manually initiate a cache refresh by invoking forceRefreshAsync.

Reply:
Fine by me, except for using the term "configuration data" instead of "config data". The latter is already used throughout the documentations. On the contrary, "configuration" is mostly used in its ordinary, "non-ConfigCat" sense.